app/main.py

```diff
@@ -1,3 +1,18 @@
-def cache(func):
-    # Write your code here
-    pass
+import functools
+
+
+def cache(func: callable) -> any:
+    stored_param_and_result = {}
+
+    @functools.wraps(func)
+    def inner(*args) -> any:
+        nonlocal stored_param_and_result
+        if args in stored_param_and_result.keys():
+            print("Getting from cache")
+            return stored_param_and_result[args]
```

Comment:
Duplicate code
[path is list[str]](https://github.com/mate-academy/py-cache-decorator/blob/master/checklist.md#dont-repeat-yourself)